dsl/src/main/java/tech/pegasys/peeps/node/genesis/GenesisFile.java

```diff
@@ -73,16 +81,18 @@ private void assertExistingGenesisMatches(final Genesis latest) {
   }
 
   private void write(final Genesis genesis) {
-    final String encodedBesuGenesis = Json.encode(genesis);
-    LOG.info(
-        "Creating Besu genesis file\n\tLocation: {} \n\tContents: {}",
-        genesisFile,
-        encodedBesuGenesis);
+    final String encodedGenesis;
+    try {
+      encodedGenesis = objectMapper.writeValueAsString(genesis);
+    } catch (JsonProcessingException e) {
```

Comment:
nit: are we enforcing final modifier in catch block i.e. `catch(final JsonProcessingException e)`

Reply:
apparently not - but nice to keep it consistent with Besu - should look into why that wasn't picked up by static checkers...